kitchen/urls.py

```diff
@@ -0,0 +1,72 @@
+from django.urls import path
+
+from .views import (
+    index,
+    DishListView,
+    DishDetailView,
+    DishCreateView,
+    DishUpdateView,
+    DishDeleteView,
+    CookListView,
+    CookDetailView,
+    CookCreateView,
+    CookYearsUpdateView,
+    CookDeleteView,
+    DishTypeListView,
+    DishTypeCreateView,
+    DishTypeUpdateView,
+    DishTypeDeleteView,
+    toggle_assign_to_dish,
+)
+
+
+urlpatterns = [
+    path("", index, name="index"),
+    path(
+        "dish-type-list/",
```

Comment:
w dalszej części mamy 
```python
path("dish/", DishListView.as_view(), name="dish-list")
```

Powinnismy być konsekwentni w nazywaniu endpointów
```suggestion
        "dish-type/",
```